### ISSUE TITLE
Publishing classes without a quiz

### ISSUE THREAD

**Issue by stevejburge** [OSTraining/OSCampus#186]
Should that be possible? Leads to confusing situations like this ...

"pass quizzes with an average of 70%" but no quiz is attached

![screen shot 2016-03-10 at 11 05 23 am](https://cloud.githubusercontent.com/assets/3868028/13675527/089a3108-e6b0-11e5-973c-40cff6e9952a.png)

**Comment by billtomczak** [comment]
Yep - early on I was informed that all we needed was the generic text

Obviously we need a bit more intelligence there :)
